src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -32,5 +32,9 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    public String getFeedbackToUser() {
```

Comment:
i think i remember telling you its ok for now, going forward, please branch properly :-)